tests/views/test_repositories.py

```diff
@@ -5,6 +5,36 @@
 
 
 class TestRepository(base.BaseCraneAPITest):
+    def test_repositories_json(self):
+        response = self.test_client.get('/crane/repositories',
+                                        headers={'Content-Type': 'application/json'})
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.headers['Content-Type'], 'application/json')
+
+        response_data = json.loads(response.data)
+        expected_data = {'baz': {'protected': True,
+                                 'tags': {'latest': 'baz123'},
+                                 'image_ids': ['baz123']},
+                         'bar': {'protected': False,
+                                 'tags': {'latest': 'def456'},
+                                 'image_ids': ['def456']},
+                         'qux': {'protected': True,
+                                 'tags': {'latest': 'qux123'},
+                                 'image_ids': ['qux123']},
+                         'redhat/foo': {'protected': False,
+                                        'tags': {'latest': 'abc123'},
+                                        'image_ids': ['abc123', 'xyz789']}}
+
+        self.assertEqual(response_data['baz'], expected_data['baz'])
+        self.assertEqual(response_data['bar'], expected_data['bar'])
+        self.assertEqual(response_data['qux'], expected_data['qux'])
+        self.assertEqual(response_data['redhat/foo'], expected_data['redhat/foo'])
+
+    def test_repositories_html(self):
```

Comment:
The HTML returned is not asserted on. Some simple ones would be good to add. We don't really have a convention for testing HTML pages, but for example ensuring all the json text is at least present on the page would be good.